README.md

```diff
@@ -1 +1,35 @@
-# HTTP_Toolkit
\ No newline at end of file
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/yx1xgac1ygxefwna/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/http_toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/HTTP_Toolkit/HTTP_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build?definitionId=67&branchName=master)
+
+# HTTP_Toolkit
+
+Description of the toolkit and any useful information or example usage. Something like : this toolkit allows interoperability between the BHoM and HTTPClient operations.
```

Comment:
> Description of the toolkit and any useful information or example usage. Something like : this toolkit allows interoperability between the BHoM and HTTPClient operations.

This still contains the explainatory template text. Needs a bit of editing.
Thanks @enarhi